mmv1/templates/terraform/examples/network_management_connectivity_test_addresses.tf.erb

```diff
@@ -1,3 +1,4 @@
+# [START networkconnectivity_test_addresses]
```

Comment:
The region tag prefix should be networkconnectivitycenter, as shown here:

https://devrel.corp.google.com/admin/products/415?t=snippets

Reply:
@betsy-lichtenberg it looks like the product is actually Network Management, not Network Connectivity Center. So I believe it should be `networkmanagement`

Reply:
I have update the tag now. Thank you @betsy-lichtenberg  @c2thorn